src/main/java/net/apnic/rdap/rdap/RDAPError.java

```diff
@@ -0,0 +1,98 @@
+package net.apnic.rdap.rdap;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class represent an error response in the RDAP protocol.
+ *
+ * @see https://tools.ietf.org/html/rfc7483
+ */
+public class RDAPError
+    extends RDAPObject
+{
+    private List<String> description = null;
```

Comment:
The way POJO deserialization is typically implemented, it requires a no argument constructor and setters. Serialization on the other hand only needs the getters. So if this message is only ever sent (serialized), you can stick final in front of all your members, initialize all the members in the constructor and ditch the setters to make your object immutable and gain all the nice properties that come with that.